tsengcoin-core/src/v1/block.rs

```diff
@@ -153,7 +153,40 @@ impl Block {
         return self.transactions;
     }
 
-    /// Like `to_txns` except it excludes the coinbase transaction.
+    pub fn get_merkle_root(&self) -> Option<Hash256> {
```

Comment:
I tested this on the genesis block with hash `0000000c9785be4989caa7cf9b7dca9161bbe8334f692fbf277fce1e23f9df2a` and got `9d00000000000000000000000000000000000000000000000000000000000000`, but the merkle root was supposed to be `e70119c836c50de74a1f6d271a0681b1cdccd3c313eb72f25b0929302e486a9d`